pyproject.toml

```diff
@@ -35,13 +34,78 @@ dependencies = [
 
 [project.optional-dependencies]
 test = [
-    "pyfakefs~=5.2",
+    "pyfakefs~=5.2"
 ]
+lint = ["ruff"]
 
 [project.urls]
 "Homepage" = "https://github.com/us-irs/cfdp-py"
 
+[tool.ruff]
+exclude = [
+    ".git",
+    "venv",
+    "docs"
+]
+line-length = 120
+target-version = "py39"
```

Comment:
are there any advantages to adding this? https://docs.astral.sh/ruff/settings/#target-version specifies this is assumed implcitely for our speicifed `requires-python`

Reply:
I have removed the target-version.